test/data/transferIncoming.json

```diff
@@ -5,7 +5,7 @@
   "amount": "1.0000",
   "data":{
     "ilp_header": {
-      "account": "ilpdemo.blue.bob.~ipr.D6-mqxYanek.22e315dc-3f99-4f89-9914-1987ceaa906d",
+      "account": "ilpdemo.blue.bob.~ipr.65f8947458feb55c.22e315dc-3f99-4f89-9914-1987ceaa906d",
```

Comment:
I think base64url is fine here. Let's be clear that ILP addresses are case-sensitive. Having a widely used module like this require case-sensitivity early on, it may actually help ensure it's consistently enforced.